README.md

```diff
@@ -138,6 +139,22 @@ Postgres.app also includes useful command line utilities (note: this list may be
 
 See [the documentation](http://postgresapp.com/documentation) for more info.
 
+## Using the debugger
+
+First, you'll need to adjust the configuration file (`postgresql.conf`) to preload the debugger extension. Find the following line and uncomment it:
+
+```
+# shared_preload_libraries = '$libdir/plugin_debugger'
```

Comment:
Our `dynamic_library_path` defaults to `$libdir`, so there should be no need to specify that.

```suggestion
# shared_preload_libraries = 'plugin_debugger'
```